src/sky.cpp

```diff
@@ -534,8 +534,12 @@ void Sky::update(float time_of_day, float time_brightness,
 	video::SColorf skycolor_bright_dawn_f = video::SColor(255, 180, 186, 250);
 	video::SColorf skycolor_bright_night_f = video::SColor(255, 0, 107, 255);
 	
-	video::SColorf cloudcolor_bright_normal_f = video::SColor(255, 240, 240, 255);
-	video::SColorf cloudcolor_bright_dawn_f = video::SColor(255, 255, 223, 191);
+	// pure white: becomes "diffuse light component" for clouds
+	video::SColorf cloudcolor_bright_normal_f = video::SColor(255, 255, 255, 255);
+	video::SColorf cloudcolor_bright_dawn_f = cloudcolor_bright_normal_f;
+	cloudcolor_bright_dawn_f.r *= 255.0f/240.0f;
+	cloudcolor_bright_dawn_f.g *= 223.0f/240.0f;
+	cloudcolor_bright_dawn_f.b *= 191.0f/255.0f;
```

Comment:
Check the original, https://github.com/minetest/minetest/blob/master/src/sky.cpp#L537 and https://github.com/minetest/minetest/blob/master/src/sky.cpp#L538 . The normal clouds were slightly blue (240,240,255), while dawn clouds were more orange-yellow (255, 223, 191). When I changed the sky to alter the cloud colors, this was to create the "dawn factor".

However, now that the sky goes with pure white clouds, the values actually don't need to be calculated. I'll use a straight assignment, and add a comment why I'm creating a color with r > 1.0